addons/edi_sale/tests/test_fail_fast.py

```diff
@@ -135,14 +129,23 @@ def test_removes_order_if_creation_fails(self):
         def create_sale(*args, **kwargs):
             cls, vals = args
             if vals["origin"] == "ORD02":
-                return ValidationError("Test Error")
+                raise ValidationError("Test Error")
             return create_method(vals)
 
+        expected_pattern = (
+            r"Failed to create for edi\.sale\.request\.tutorial\.record\(\d+,\), ORD02"
+        )
+
         doc = self.create_tutorial("order05.csv", fail_fast=False)
         with mock.patch.object(
             Sale.__class__, "create", create=True, autospec=True, side_effect=create_sale
         ):
-            self.assertTrue(doc.action_execute())
+            with self.assertLogs(level="ERROR") as cm:
```

Comment:
I'd include the logger here, so 
```python
with self.assertLogs('odoo.addons.odoo_edi.models.edi_synchronizer', level="ERROR"):
```
(double-check the logger name, I might have got it wrong).

Do the same in the other test too.